menu.md

```diff
@@ -1,62 +1,62 @@
-- [Slots](/docs/(qwik)/components/slots/index.mdx)
-- [Rendering](/docs/(qwik)/components/rendering/index.mdx)
-- [Styling](/docs/(qwik)/components/styles/index.mdx)
+- [Visión general](/docs/(qwik)/components/overview/index.mdx)
+- [Estado](/docs/(qwik)/componentes/estado/index.mdx)
+- [Eventos](/docs/(qwik)/components/events/index.mdx)
+- [Tareas y ciclo de vida](/docs/(qwik)/components/tasks/index.mdx)
+- [Contexto](/docs/(qwik)/componentes/contexto/index.mdx)
+- [Ranuras (Slots)](/docs/(qwik)/components/slots/index.mdx)
+- [Renderización](/docs/(qwik)/components/rendering/index.mdx)
+- [Estilos](/docs/(qwik)/components/styles/index.mdx)
 
-## Routing (Qwik City)
+## Enrutamiento (Qwik City)
 
 - [Qwik City](/docs/(qwikcity)/qwikcity/index.mdx)
-- [Routing](/docs/(qwikcity)/routing/index.mdx)
-- [Route Loaders](/docs/(qwikcity)/route-loader/index.mdx)
-- [Form actions](/docs/(qwikcity)/action/index.mdx)
+- [Rutas](/docs/(qwikcity)/rutas/index.mdx)
+- [Cargadores de ruta](/docs/(qwikcity)/cargador-de-ruta/index.mdx)
+- [Acciones de formulario](/docs/(qwikcity)/action/index.mdx)
 - [Layouts](/docs/(qwikcity)/layout/index.mdx)
-- [Pages](/docs/(qwikcity)/pages/index.mdx)
+- [Páginas](/docs/(qwikcity)/pages/index.mdx)
 - [Endpoints](/docs/(qwikcity)/endpoints/index.mdx)
-- [server$](/docs/(qwikcity)/server$/index.mdx)
-- [Env variables](/docs/(qwikcity)/env-variables/index.mdx)
-- [API reference](/docs/(qwikcity)/api/index.mdx)
-
-## Integrations
-
-- [Overview](integrations/index.mdx)
-- [Auth.js](integrations/authjs/index.mdx)
-- [Builder.io](integrations/builderio/index.mdx)
-- [Cypress](integrations/cypress/index.mdx)
-- [Image Optimization](integrations/image-optimization/index.mdx)
-- [Icons](integrations/icons/index.mdx)
-- [Internationalization](integrations/i18n/index.mdx)
-- [Modular Forms](integrations/modular-forms/index.mdx)
+- [server$](/docs/(qwikcity)/servidor$/index.mdx)
+- [Variables de Entorno](/docs/(qwikcity)/env-variables/index.mdx)
+- [Referencia API](/docs/(qwikcity)/api/index.mdx)
+
+## Integraciones
+
+- [Resumen](integraciones/index.mdx)
+- [Auth.js](integraciones/authjs/index.mdx)
+- [Builder.io](integraciones/builderio/index.mdx)
+- [Cypress](integraciones/cypress/index.mdx)
+- [Optimización de imágenes](integrations/image-optimization/index.mdx)
+- [Iconos](integrations/icons/index.mdx)
+- [Internacionalización](integrations/i18n/index.mdx)
+- [Formularios modulares](integrations/modular-forms/index.mdx)
 - [Nx Monorepos](integrations/nx/index.mdx)
 - [Partytown](integrations/partytown/index.mdx)
 - [Playwright](integrations/playwright/index.mdx)
 - [PostCSS](integrations/postcss/index.mdx)
 - [Prisma](integrations/prisma/index.mdx)
 - [React](integrations/react/index.mdx)
 - [Storybook](integrations/storybook/index.mdx)
-- [Styled Vanilla Extract](integrations/styled-vanilla-extract/index.mdx)
+- [Vanilla Estilizado](integrations/styled-vanilla-extract/index.mdx)
```

Comment:
_extract_ ??

Reply:
@almilo @danywalls @da8ah @manuelsanchez2 @leifermendez 

¿Qué pondríais en este caso?

Gracias :)

Reply:
Yo opino dejarlo igual dado que según la [página](https://qwik.builder.io/docs/integrations/styled-vanilla-extract/) se trata de un framework con ese nombre, traducirlo quizá estaría demás al ser nombre propio.

Reply:
Style Vanilla dejaría yo

Reply:
Viendo que opináis lo mismo, vamos a ponerlo con el el nombre original 👍